gnovm/tests/files/zrealm_crossrealm22a.gno

```diff
@@ -0,0 +1,29 @@
+// PKGPATH: gno.land/r/crossrealm_test
+package crossrealm_test
+
+import (
+	crossrealm "gno.land/r/demo/tests/crossrealm/func"
+)
+
+var b = 1
+
+type Local_S struct {
+	name string
+}
+
+func main() {
+	f := func() bool {
+		c := b
+		var ls Local_S
+		return true
+	}
+	crossrealm.SetCallback(f)
+	println("ok")
+}
+
+// Output:
+// ok
+
+// Realm:
+// switchrealm["gno.land/r/demo/tests/crossrealm/func"]
+// switchrealm["gno.land/r/crossrealm_test"]
```

Comment:
please see this discussion about `FuncValue` association .
https://github.com/gnolang/gno/pull/2958#discussion_r1939204053